hoomd/hpmc/UpdaterRemoveDrift.h

```diff
@@ -113,8 +155,8 @@ template<class Shape> class RemoveDriftUpdater : public Updater
         }
 
     protected:
-    std::shared_ptr<ExternalFieldLattice<Shape>> m_externalLattice;
     std::shared_ptr<IntegratorHPMCMono<Shape>> m_mc;
+    std::vector<vec3<Scalar>> m_ref_positions;
```

Comment:
I agree. In that case, should we consider removing the Shape template altogether and, perhaps, moving the updater to a different namespace, like `hoomd.update.RemoveDrift`?

Reply:
@b-butler what do you think?

Reply:
I would say if there is any utility to the updater in MD then use the more general namespace (I am not sure there is), and if not leave it in HPMC. If we were to add a new MC move type that did not directly rely on the integrator, we would still put that under `hoomd.hpmc` (I think), so I feel like other updaters should be the same.

However, if MD could use the updater productively in some cases, then the more general namespace should be used for visibility since many people who use `hoomd.md` may never use `hoomd.hpmc`.

Reply:
This method is to improve the accuracy of FrenkelLladd calculations. Those can be performed in MD as well as MC, so there is at least one case where a user may want to use it outside of HPMC.

Reply:
Since it doesn't require integrator information and there is at least one use case, I believe it should be placed a more general namespace. However, I'll wait for consensus before proceeding. On another note: I have already removed the shape template and have addressed a bunch of the open comments locally.

Reply:
Okay, then we should move to the more general namespace.

Reply:
Sounds good. I'll work on it.